sources/zsh/history

```diff
@@ -351,4 +351,5 @@ kerbrute userenum -d BREAKING.BAD usernames.txt
 kerbrute passwordspray -d BREAKING.BAD domain_users.txt Password123
 kerbrute bruteuser -d BREAKING.BAD passwords.lst thoffman
 tls-scanner -connect example.com:443
+hekatomb.py -hashes :ed0052e5a66b1c8e942cc9481a50d56 DOMAIN.local/administrator@10.0.0.1 -debug -dnstcp
```

Comment:
Can you make sure the values are replaced to what the history now features since https://github.com/ShutdownRepo/Exegol-images/commit/b7ff3742d2bc9ae5c7f9ad9370387fac9c06e0a0 ?

Reply:
Resolved in second commit ;)